nixpkgs_review/cli/__init__.py

```diff
@@ -188,6 +189,13 @@ def common_flags() -> List[CommonFlag]:
             default="",
             help="Passed to nix-shell to run a command instead of an interactive nix-shell",
         ),
+        CommonFlag(
+            "--sandbox",
+            action="store_true",
+            help="Wraps nix-shell inside a sandbox (requires `bwrap` in PATH)",
+        )
+        if platform.system() == "Linux"
+        else None,
```

Comment:
I don't like the idea that the flag is hidden on other platforms. You should not be able to enable it on darwin but it should still show up in --help IMO.